repo/packages/B/beta-tensorflow/0/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "packagingVersion": "4.0",
+    "minDcosReleaseVersion": "1.9",
+    "upgradesFrom": [
+    ],
+    "downgradesTo": [
+    ],
+    "name": "beta-tensorflow",
+    "version": "2.0.2-1.3.0-beta",
+    "maintainer": "dcos.io/community/",
+    "description": "TensorFlow on DC/OS",
+    "selected": false,
+    "framework": true,
+    "tags": [
+        "tensorflow"
+    ],
+    "preInstallNotes": "This DC/OS Service is currently a beta candidate undergoing testing as part of a formal beta test program.\n\nThere may be bugs, incomplete features, incorrect documentation, or other discrepancies.",
```

Comment:
This isn't a part of a formal beta program is it? My understanding is it was a community package.

Reply:
What is the wording for community packages?

Reply:
@joerg84 can you provide some test here instead of the "formal beta program" blurb?

Reply:
I have updated the text to read: `"preInstallNotes": "This is a community package. Community packages are unverified and unreviewed content from the community.",`. This seems to be in keeping with some of the other community packages.